mythril/interfaces/cli.py

```diff
@@ -197,13 +197,20 @@ def create_parser(parser: argparse.ArgumentParser) -> None:
         default=50,
         help="Maximum recursion depth for symbolic execution",
     )
-
     options.add_argument(
         "--strategy",
         choices=["dfs", "bfs", "naive-random", "weighted-random"],
         default="bfs",
         help="Symbolic execution strategy",
     )
+    options.add_argument(
+        "-b",
+        "--loop-bound",
+        type=int,
+        default=4,
+        help="Bound loops at n iterations",
+        metavar="N",
```

Comment:
Should we have metavar as N here? as that would be used for display of the arg.We can just skip it.

Reply:
I thought it would be cool to say "n iterations" and call the metavar N 😀 Can we leave it like that?